tests/unit/test_submit.py

```diff
@@ -0,0 +1,69 @@
+"""
+Testing the submit function in __main__.py
+"""
+
+import os
+from unittest.mock import Mock
+
+import pytest
+from armada_client.client import ArmadaClient
+from armada_client.typings import EventType
+from armada_jupyter.__main__ import submit_worker
+
+TEST_FILE = "tests/files/general.yml"
+JOB_ID = "test_job_id"
+
+# set variables in environment
+os.environ["ARMADA_HOST"] = ""
+os.environ["ARMADA_PORT"] = ""
+os.environ["DISABLE_SSL"] = "True"
+
+
+class FakeArmadaClient(ArmadaClient):
```

Comment:
There is a lot of repetition in objects. Do you guys think it makes sense to have seperate file that stores the test objects?

Reply:
I think this is fine.